packages/theming/apple-theme/src/appleTheme.colors.macos.ts

```diff
@@ -0,0 +1,223 @@
+    buttonFocusedBackground: fluentApple.communicationBlue,
+    buttonFocusedBorder: 'transparent',
+    buttonFocusedContent: appleSemantic.controlTextColor,
+    buttonFocusedIcon: appleSemantic.controlTextColor,
+
+    // TODO Native Module for withSystemEffect(.pressed)
+    buttonPressedBackground: fluentApple.communicationBlueTint10,
+    buttonPressedBorder: 'transparent',
+    buttonPressedContent: appleSemantic.controlTextColor,
+    buttonPressedIcon: appleSemantic.controlTextColor,
+
+    // TODO Native Module for withSystemEffect(.disabled)
+    buttonDisabledBackground: fluentApple.gray100,
+    buttonDisabledBorder: 'transparent',
+    buttonDisabledContent: appleSemantic.controlTextColor,
+    buttonDisabledIcon: appleSemantic.controlTextColor,
+
+    ghostBackground: 'transparent',
+    ghostBorder: 'transparent',
+    ghostContent: appleSemantic.controlTextColor,
+    ghostIcon: appleSemantic.controlTextColor,
+
+    ghostHoveredBackground: 'transparent',
+    ghostHoveredBorder: 'transparent',
+    ghostHoveredContent: appleSemantic.controlTextColor,
+    ghostHoveredIcon: appleSemantic.controlTextColor,
+
+    ghostFocusedBackground: 'transparent',
+    ghostFocusedBorder: 'transparent',
+    ghostFocusedContent: appleSemantic.controlTextColor,
+    ghostFocusedIcon: appleSemantic.controlTextColor,
+
+    // TODO System Effect(.pressed)
+    ghostPressedBackground: 'transparent',
+    ghostPressedBorder: 'transparent',
+    ghostPressedContent: fluentApple.communicationBlueTint20,
+    ghostPressedIcon: fluentApple.communicationBlueTint20,
+
+    // TODO Native Module for withSystemEffect(.disabled)
+    ghostDisabledBackground: 'transparent',
+    ghostDisabledBorder: 'transparent',
+    ghostDisabledContent: fluentApple.communicationBlueTint20,
+    ghostDisabledIcon: fluentApple.communicationBlueTint20,
+
+    brandBackground: fluentApple.communicationBlue,
+    brandBorder: 'transparent',
+    brandContent: appleSemantic.controlTextColor,
+    brandIcon: appleSemantic.controlTextColor,
+
+    brandHoveredBackground: fluentApple.communicationBlue,
+    brandHoveredBorder: 'transparent',
+    brandHoveredContent: appleSemantic.controlTextColor,
+    brandHoveredIcon: appleSemantic.controlTextColor,
+
+    brandFocusedBackground: fluentApple.communicationBlue,
+    brandFocusedBorder: 'transparent',
+    brandFocusedContent: appleSemantic.controlTextColor,
+    brandFocusedIcon: appleSemantic.controlTextColor,
+
+    brandPressedBackground: fluentApple.communicationBlueTint10,
+    brandPressedBorder: 'transparent',
+    brandPressedContent: appleSemantic.controlTextColor,
+    brandPressedIcon: appleSemantic.controlTextColor,
+
+    // TODO System Effect
+    brandDisabledBackground: fluentApple.gray100,
+    brandDisabledBorder: 'transparent',
+    brandDisabledContent: appleSemantic.controlTextColor,
+    brandDisabledIcon: appleSemantic.controlTextColor,
+
+    // TODO What is checked?
```

Comment:
Checkbox / toggle button?

Reply:
Yes. I made the assumption that would match the "pressed" state, but would love to hear other opinions from you or @harrieshin or designers or whoever else